.nf-core.yml

```
@@ -1,5 +1,16 @@
-repository_type: pipeline
-
 lint:
   files_exist:
     - conf/igenomes.config
+  nextflow_config:
+    - params.input
+nf_core_version: 3.2.0
+repository_type: pipeline
+template:
+  author: Daniel Lundin
+  description: Performs phylogenetic placement with EPA-NG
+  force: false
+  is_nfcore: true
+  name: phyloplace
+  org: nf-core
+  outdir: .
```

```suggestion
  outdir: .
  skip_features:
    - igenomes
```
will make things easier in the future https://nf-co.re/docs/nf-core-tools/pipelines/create#customization